Sources/Redis.swift

```diff
@@ -1505,115 +1546,884 @@ public class Redis {
+    ///
+    /// - Parameter destination: the destination of the result
+    /// - Parameter keys: a variadic parameter of the keys to intersect from
+    ///
+    /// Returns: Integer reply: the number of elements in the resulting set.
+    ///
+    public func sinterstore(_ destination: RedisString, keys: RedisString..., callback: (Int?, NSError?) -> Void) {
+        self.sinterstoreArrayOfKeys(destination, keys: keys, callback: callback)
+    }
+    
+    ///
+    /// Intersect multiple sets and store the resulting set in a key
+    ///
+    /// - Parameter destination: the destination of the result
+    /// - Parameter keys: an array parameter of the keys to intersect from
+    ///
+    /// Returns: Integer reply: the number of elements in the resulting set.
+    ///
+    public func sinterstoreArrayOfKeys(_ destination: RedisString, keys: [RedisString], callback: (Int?, NSError?) -> Void) {
+        var command = [RedisString("SINTERSTORE"), destination]
+        for key in keys {
+            command.append(key)
+        }
+        issueCommandInArray(command) {(response: RedisResponse) in
+            self.redisIntegerResponseHandler(response, callback: callback)
+        }
+    }
+    
+    ///
+    /// Determine if a given value is a member of a set
+    ///
+    /// - Parameter key: the String paramter for the key
+    /// - Parameter member: the String paramter for the member
+    ///
+    /// Returns: Integer reply: 1 if element is a member of the set,
+    ///                         0 if the element isn't a member of the set, or if key doesn't exist.
+    ///
+    public func sismember(_ key: String, member: String, callback: (Int?, NSError?) -> Void) {
+        issueCommand("SISMEMBER", key, member) {(response: RedisResponse) in
+            self.redisIntegerResponseHandler(response, callback: callback)
+        }
+    }
+    
+    ///
+    /// Determine if a given value is a member of a set
+    ///
+    /// - Parameter key: the RedisString paramter for the key
+    /// - Parameter member: the RedisString paramter for the member
+    ///
+    /// Returns: Integer reply: 1 if element is a member of the set,
+    ///                         0 if the element isn't a member of the set, or if key doesn't exist.
+    ///
+    public func sismember(_ key: RedisString, member: RedisString, callback: (Int?, NSError?) -> Void) {
+        issueCommand(RedisString("SISMEMBER"), key, member) {(response: RedisResponse) in
+            self.redisIntegerResponseHandler(response, callback: callback)
+        }
+    }
+    
+    ///
+    /// Move a member from one set to another
+    ///
+    /// - Parameter source: the Source set from where to move the member from
+    /// - Parameter destination: the Destination set from where to move the member to
+    /// - Parameter member: the String parameter for the member to be moved
+    ///
+    /// Returns: Integer reply: 1 if element is moved,
```

Comment:
I think you should return a Bool not an Int. There is a redisBoolResponseHanlder that should take an Integer result and call a callback that takes a Bool.